src/i18n/en.js

```diff
@@ -56,6 +57,19 @@ const messages_en = {
     volume_500: 'Between 500k and 1m',
     volume_do_not_know: `I don't know`,
   },
+  big_query: {
+    free_btn_redirect: `Get to know our Plus plan`,
+    free_text_data_studio_one: `Learn to configure our dashboard`,
```

Comment:
<pre>
free_text_data_studio: `Learn to configure our dashboard [Data Studio in a few steps]({$urlUpgradePlan}) , or access BigQuery through another data visualization tool such as Tableau.
`,
</pre>

and this the content in 1 resource with markdown